modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysEdgeCefTest.ts

```diff
@@ -0,0 +1,282 @@
+import { Keys } from '@ephox/agar';
+import { describe, it, context } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import * as CaretContainer from 'tinymce/core/caret/CaretContainer';
+
+describe('browser.tinymce.core.keyboard.ArrowKeysEdgeCefTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, [], true);
+
+  const assertStartContainer = (editor: Editor, f: (node: Node) => boolean) => {
+    const startContainer = editor.selection.getRng().startContainer;
+    assert.isTrue(f(startContainer), 'Check selection is in caret container');
+  };
+
+  context('Cmd+Shift+Up/Down should expand selection to the edge of the content', () => {
+    it('TINY-7795: Cmd+Shift+Up when cef block at the start', () => {
```

Comment:
```suggestion
    it('TINY-7795: Cmd+Shift+Up when a cef block is at the start', () => {
```

Maybe add the `a` and `is` on all of them. Not sure. Maybe the native English people has better grammar ideas. :)